### ISSUE TITLE
SET_VEHICLE_FUEL_LEVEL: No such entity

### ISSUE THREAD

**Issue by GiNside97** [InZidiuZ/LegacyFuel#91]
Using function: exports["LegacyFuel"]:SetFuel(vehicle, 100)

But getting back:
SET_VEHICLE_FUEL_LEVEL: No such entity

What Can I do?

Structure of the File:

![image](https://user-images.githubusercontent.com/49701274/187929116-9ff3fbaa-077e-4728-a750-24102a52c513.png)

![image](https://user-images.githubusercontent.com/49701274/187929452-ed038b3d-2d2a-4128-b54c-ad3ed8e06f5c.png)

**Comment by klose2410** [comment]
i have same fault after update...

**Comment by W1Woo** [comment]
someone figure it out ?

**Comment by rockstarrepublic1** [comment]
> someone figure it out ?

No. Replace with cc-fuel or cdn-fuel. cdn-fuel is a bit more work, but has more options like electric vehicle charging.
